src/main/java/com/google/cloud/genomics/dataflow/pipelines/IdentityByState.java

```diff
@@ -121,10 +121,10 @@ public static void main(String[] args) throws Exception {
             new JoinNonVariantSegmentsWithVariants.RetrieveAndCombineTransform(auth, VARIANT_FIELDS));
       } else {
         processedVariants = requests.apply(
-            new VariantStreamer(auth, ShardBoundary.Requirement.NON_VARIANT_OVERLAPS, VARIANT_FIELDS));
+            new VariantStreamer(auth, ShardBoundary.Requirement.STRICT, VARIANT_FIELDS));
       }
     } else {
-      // Computing IBS over genomic region(s) or the whole genome.
+      // Compute IBS over genomic region(s) or the whole genome.
```

Comment:
Proposed substitution for the whole sentence:

*Computes IBS over one or more genomic regions, or the entire genome.*

Feel free to use *whole* instead of *entire* if that is preferred.